code/modules/client/client_procs.dm

```diff
@@ -152,9 +152,6 @@ GLOBAL_LIST_INIT(blacklisted_builds, list(
 GLOBAL_LIST_EMPTY(external_rsc_urls)
 #endif
 
-/client/can_vv_get(var_name)
-	return var_name != NAMEOF(src, holder) && ..()
-
 /client/vv_edit_var(var_name, var_value)
 	return var_name != NAMEOF(src, holder) && ..()
```

Comment:
You still need to get rid of this as client already has a vv_edit_var override that checks for holder. remove the `closes` on my first pr until you do.